src/components/metrics/modal/index.js

```diff
@@ -109,6 +123,21 @@ export class MetricModal extends Component {
                   size={'large'}
                 />
               </div>
+              <div className='col-sm-2'>
+
+              <div className='metricModal--sample-container'>
+                <DropDown
+                  headerText={'Samples'}
+                  openLink={<a className='modal-action button'> Samples </a>}
+                  position='right'
+                  hasPadding={true}
+                  ref='DropDown'
```

Comment:
Tiny thing, but can we not add a `ref` here? Sometimes `ref`s cause problems and this one doesn't appear used, is all.